Makefile

```diff
@@ -2320,6 +2320,7 @@ else
 	oc apply -f ${OPERATOR_DIR}
 	timeout ${TIMEOUT} bash -c "while ! (oc get deployments/nmstate-operator -n ${NAMESPACE}); do sleep 10; done"
 	oc wait deployments/nmstate-operator -n ${NAMESPACE} --for condition=Available --timeout=${TIMEOUT}
+	timeout ${TIMEOUT} bash -c "while ! (oc wait pod -n openshift-apiserver -l apiserver=true --for condition=Ready); do sleep 10; done"
```

Comment:
500s might not be enough to establish cluster after etcd crash, but let's try.

Reply:
with etcd crash i think we will not be able to deploy other things too, i just used the same timeout like other places.